dotCMS/src/main/java/com/dotcms/rest/api/v1/system/redis/RedisResource.java

```diff
@@ -0,0 +1,250 @@
+package com.dotcms.rest.api.v1.system.redis;
+
+import com.dotcms.cache.lettuce.RedisClient;
+import com.dotcms.cache.lettuce.RedisClientFactory;
+import com.dotcms.rest.ResponseEntityView;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.dotcms.rest.api.v1.authentication.ResponseUtil;
+import com.dotcms.util.CollectionsUtils;
+import com.dotmarketing.util.PortletID;
+import com.dotmarketing.util.UtilMethods;
+import com.google.common.annotations.VisibleForTesting;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.Suspended;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Set;
+import java.util.concurrent.Future;
+
+/**
+ * Redis Resource
+ * @author jsanca
+ */
+@Path("/v1/redis")
+public class RedisResource {
```

Comment:
This resource should have postman tests

Reply:
will mark visible for testing, bc we do not want to share this one